5/README.md

```diff
@@ -22,7 +26,12 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Tango Property Specification
 
- A Property is designed to represent any information of configuration in Tango.
+ A Property is a persisted item of Tango that is mainly used for configuration purposes. 4 types of properties exist:
+
+ *  Class property: a property that is attached to a Device Class (cf RFC-9). This property is accessible from all devices of the class
+ *  Device property: a property that is attached to a Device (RFC-2). This property is accessible from only one device
```

Comment:
> This property is accessible from all devices of the class
> This property is accessible from only one device
I find these sentences a bit confusing... Technically, nothing prevents another device or a client to have access to a device property value defined in the DB or even to the default value of a device property.